packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -39,9 +39,10 @@ typedef DragTargetAcceptWithDetails<T> = void Function(DragTargetDetails<T> deta
 /// Signature for building children of a [DragTarget].
 ///
 /// The `candidateData` argument contains the list of drag data that is hovering
-/// over this [DragTarget] and that has passed [DragTarget.onWillAccept]. The
-/// `rejectedData` argument contains the list of drag data that is hovering over
-/// this [DragTarget] and that will not be accepted by the [DragTarget].
+/// over this [DragTarget] and that has passed
```

Comment:
Can we update the comment as below :

/// The `candidateData` argument contains the list of drag data that is hovering
/// over this [DragTarget] and that has passed [DragTarget.onWillAcceptWithDetails].
/// The `rejectedData` argument contains the list of drag data that is hovering over this 
/// [DragTarget] and that will not be accepted by the [DragTarget].

Reply:
Did you just reformat it or am I missing what the change is?

Reply:
I changed one mention of (deprecated) DragTarget.onWillAccept to DragTarget.onWillAcceptWithDetails and re-wrapped.